File: app/src/main/java/com/brainwellnessspa/DashboardTwoModule/elevate/ElevateFragment.java
package com.brainwellnessspa.DashboardTwoModule.elevate;

import android.os.Bundle;

import androidx.databinding.DataBindingUtil;
import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.brainwellnessspa.R;
import com.brainwellnessspa.databinding.FragmentElevateBinding;

public class ElevateFragment extends Fragment {
    FragmentElevateBinding binding;

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        binding = DataBindingUtil.inflate(inflater, R.layout.fragment_elevate, container, false);
        View view = binding.getRoot();
        return view;
    }
}

File: app/src/main/java/com/brainwellnessspa/dashboardModule/elevate/ElevateFragment.kt
package com.brainwellnessspa.dashboardModule.elevate

import android.app.Activity
import android.content.Context
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.databinding.DataBindingUtil
import androidx.fragment.app.Fragment
import com.brainwellnessspa.BWSApplication
import com.brainwellnessspa.R
import com.brainwellnessspa.databinding.FragmentElevateBinding

class ElevateFragment : Fragment() {
    lateinit var binding: FragmentElevateBinding
    lateinit var ctx: Context
    lateinit var act: Activity

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
        binding = DataBindingUtil.inflate(inflater, R.layout.fragment_elevate, container, false)
        val view = binding.root
        ctx = requireActivity()
        act = requireActivity()
        networkCheck()

        return view
    }

    override fun onResume() {
        networkCheck()
        super.onResume()
    }

    private fun networkCheck() {
        if (BWSApplication.isNetworkConnected(act)) {
            binding.llRemainDev.visibility = View.VISIBLE
            binding.llNoInternet.visibility = View.GONE
        } else {
            binding.llRemainDev.visibility = View.GONE
            binding.llNoInternet.visibility = View.VISIBLE
        }
    }
}